### PR TITLE
chore: move dependencies into dev dependencies

### PR DESCRIPTION
**Pull request by fredlacs**

### CODE REVIEW

**Comment by fredlacs** [comment]
breaking change on network type to remove rpc url

File: scripts/instantiate_bridge.ts [hunk 1]
```diff
@@ -85,14 +85,20 @@ export const instantiateBridge = (
...
-  if (!l1Network.rpcURL) {
-    throw new Error('L1 rpc url not set (see .env.sample or networks.ts)')
-  }
-  if (!l2Network.rpcURL) {
-    throw new Error('L2 rpc url not set (see .env.sample or utils/networks.ts)')
-  }
-  const ethProvider = new JsonRpcProvider(l1Network.rpcURL)
-  const arbProvider = new JsonRpcProvider(l2Network.rpcURL)
+  const l1Rpc = (() => {
+    if (networkID === 42161) return process.env['MAINNET_RPC'] as string
```
Comment (yahgwai) [inline scripts/instantiate_bridge.ts:89]: network id is always l2 network - could rename it to l2networkId

File: src/lib/dataEntities/networks.ts [hunk 2]
```diff
@@ -135,7 +131,6 @@ export const l1Networks: L1Networks = {
     explorerUrl: 'https://etherscan.io',
     partnerChainIDs: [42161],
     blockTime: 14,
-    rpcURL: process.env['MAINNET_RPC'] as string,
```
Comment (yahgwai) [inline src/lib/dataEntities/networks.ts:138]: I've got a feeling these are used in the token-bridge-ui, we should take a look there

File: scripts/instantiate_bridge.ts [hunk 3]
```diff
@@ -85,14 +85,20 @@ export const instantiateBridge = (
...
+    if (networkID === 421611) return process.env['RINKEBY_RPC'] as string
+    if (networkID === 1338) return 'http://127.0.0.1:8545/'
```
Comment (yahgwai) [inline scripts/instantiate_bridge.ts:91]: What will we do for unknown networks, like custom networks, local networks on different chain id or newly created devnets?

File: src/lib/dataEntities/networks.ts (hunk 2 above)
Comment (fredlacs) [reply src/lib/dataEntities/networks.ts:138]: yea but to instantiate a network you give a provider (in getNetwork)
so they already have the rpc lying around

File: scripts/instantiate_bridge.ts (hunk 3 above)
Comment (fredlacs) [reply scripts/instantiate_bridge.ts:91]: they can have their own `instantiate_bridge` script or we can add support when needed.
this script isn't published as part of the package, its a dev utility

File: src/lib/dataEntities/networks.ts (hunk 2 above)
Comment (fredlacs) [reply src/lib/dataEntities/networks.ts:138]: I don't think its useful for us to have the rpc url here since its never used by the sdk package itself

Comment (fredlacs) [reply src/lib/dataEntities/networks.ts:138]: we should be good
https://github.com/OffchainLabs/arb-token-bridge/search?q=rpcURL

Comment (fredlacs) [reply src/lib/dataEntities/networks.ts:138]: cc @spsjvc

**Review by yahgwai — APPROVED** [review]

**Pull request merged**